interop-tests/src/test/scala/akka/grpc/scaladsl/GrpcExceptionHandlerSpec.scala

```diff
@@ -114,6 +137,8 @@ class GrpcExceptionHandlerSpec
       statusHeader.map(_.value()) should be(Some("3"))
       val statusMessageHeader = lastChunk.trailer.find { _.name == "grpc-message" }
       statusMessageHeader.map(_.value()) should be(Some("No name found"))
+      val richErrorHeader = lastChunk.trailer.find { _.name == RichErrorHeader.name }
```

Comment:
I think if you make it a `case class` you could even:

```
val richErrorHeaderValue = lastChunk.trailer.collectFirst { case RichErrorHeader(value) => value }
richErrorHeaderValue should be(Some("test-data"))
```
Haven't tested though.

In any case, nice test!

Reply:
Done!